downloads/id/github-git-cheat-sheet.md

```diff
@@ -0,0 +1,215 @@
+
+Batal merevisi berkas, namun tetap mempertahankan isinya
+
+```$ git commit -m "[descriptive message]"```
+
+Daftarkan perubahan berkas secara permanen di riwayat revisi
+
+## Perubahan untuk banyak berkas 
+Namai serangkaian _commit_ dan gabungkan upaya pengubahan jadi satu
+
+```$ git branch```
+
+Daftar semua cabang lokal di repositori saat ini
+
+```$ git branch [nama-cabang]```
+
+Buat cabang baru
+
+```$ git checkout [nama-cabang]```
+
+Berpindah ke cabang tertentu dan perbarui direktori yang sedang dikerjakan
+
+```$ git merge [nama-cabang]```
+
+Menggabungkan riwayat cabang tertentu ke dalam cabang yang sedang dipakai
+
+```$ git branch -d [nama-cabang]```
+
+Hapus cabang tertentu
+{% endcapture %}
+<div class="col-md-6">
+{{ colTwo | markdownify }}
+</div>
+<div class="clearfix"></div>
+
+
+{% capture colThree %}
+## Pergantian nama berkas
+Relokasi dan menghapus berkas terevisi
+
+```$ git rm [berkas]```
+
+Menghapus berkas dari direktori kerja dan daftarkan penghapusan
+
+```$ git rm --cached [berkas]```
+
+Menghapus berkas dari riwayat revisi dengan tetap mempertahankan berkas lokal
+
+```$ git mv [berkas-asli] [berkas-baru]```
+
+Mengganti nama berkas dan mempersiapkan berkas untuk pendaftaran _commit_
+
+## Tahan pelacakan
+Mengabaikan berkas dan jalur untuk sementara
+
+```
+*.log
+build/
+temp-*
+```
+
+Sebuah berkas teks bernama `.gitignore` mengabaikan revisi berkas yang tidak disengaja serta jalur berkas yang cocok dengan pola tertentu
+
+```$ git ls-files --other --ignored --exclude-standard```
+
+Daftarkan semua berkas yang diabaikan dalam proyek ini
+
+## Simpan fragmen
+Menyimpan dan mengembalikan perubahan yang belum lengkap
+
+```$ git stash```
+
+Menyimpan semua perubahan berkas yang terlacak untuk sementara
+
+```$ git stash pop```
+
+Mengembalikan berkas yang paling baru disimpan
+
+```$ git stash list```
+
+Daftar semua koleksi perubahan yang tersimpan
+
+```$ git stash drop```
+
+Membuang koleksi perubahan yang paling baru disimpan
+{% endcapture %}
+<div class="col-md-6">
+{{ colThree | markdownify }}
+</div>
+
+{% capture colFour %}
+## riwayat ulasan
+Jelajah dan periksa perkembangan berkas-berkas dalam proyek
+
+```$ git log```
+
+Daftar riwayat revisi untuk cabang saat ini
+
+```$ git log --follow [berkas]```
+
+Daftar riwayat revisi untuk sebuah berkas, termasuk pergantian namanya
+
+```$ git diff [cabang-pertama]...[cabang-kedua]```
+
+Menunjukkan perbedaan konten antar dua cabang
+
+`` `$ git show [commit]` ``
+
+Mennampilkan perubahan konten dan metadata yang terdaftar dalam sebuah komit
+
+## Lakukan _commit_ kembali
+Menghapus kesalahan dan buat riwayat penggantian
+
+`` `$ git reset [komit]` ``
+
+Membatalkan semua _commit_ setelah `[commit]`, dengan melestarikan perubahan lokal
+
+`` `$ Git reset --hard [commit]` ``
+
+Membuang semua riwayat dan perubahan sampai di titik yang ditentukan oleh _commit_
+
+## Sinkronisasi perubahan
+Daftar (tautan) dura dan tukar riwayat repositori
```

Comment:
Well.. `dura` is [a valid word in kbbi](http://kbbi.web.id/dura-2). But I think it's a bit too.. uncommon? Is there any alternative for the term `remote` in Indonesian?